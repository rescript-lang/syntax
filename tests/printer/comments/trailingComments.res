let user = {
  name: "steve",
  age: 31 /* here */, // test 
  
  /* test */
  /* test2 */

  // test3 

  /* test 4 */
}

/* A */ /*
* A type that can be written to a buffer.
*/ /*
* Describes the connection status of a ReactiveSocket/DuplexConnection.
* - NOT_CONNECTED: no connection established or pending.
* - CONNECTING: when `connect()` has been called but a connection is not yet
*   established.
* - CONNECTED: when a connection is established.
* - CLOSED: when the connection has been explicitly closed via `close()`.
* - ERROR: when the connection has been closed for any other reason.
*/ /*
* A contract providing different interaction models per the [ReactiveSocket protocol]
* (https://github.com/ReactiveSocket/reactivesocket/blob/master/Protocol.md).
*/ /*
* A single unit of data exchanged between the peers of a `ReactiveSocket`.
*/
