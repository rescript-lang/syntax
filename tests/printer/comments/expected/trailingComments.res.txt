
  Syntax error!
  tests/printer/comments/trailingComments.res:13:9-15:2

  11 │ }
  12 │ 
  13 │ /* A */ /**
  14 │ * A type that can be written to a buffer.
  15 │ */ /**
  16 │ * Describes the connection status of a ReactiveSocket/DuplexConnection.
  17 │ * - NOT_CONNECTED: no connection established or pending.

  Did you forget to attach `ns.doc` to an item?
  Standalone attributes start with `@@` like: `@@ns.doc`

